app/app.go

```diff
@@ -87,7 +87,7 @@ func commands(defaultCfg *sconfig.Config) []*cli.Command {
 	return append(
 		append(
 			serverCommands(defaultCfg),
-			common.WithFlags(clientCommands, common.SharedFlags)...,
+			common.WithFlags(clientCommands, common.FlagsForPaginationAndRendering)...,
```

Comment:
why changing this? 

Pagination And Rendering flags apply to some commands, not all client commands